jwst/jump/jump_step.py

```diff
@@ -45,13 +48,15 @@ def process(self, input):
             # Retrieve the parameter values
             rej_thresh = self.rejection_threshold
             max_cores = self.maximum_cores
+            max_jump_to_flag_neighbors = self.max_jump_to_flag_neighbors
+            min_jump_to_flag_neighbors = self.min_jump_to_flag_neighbors
+            flag_4_neighbors = self.flag_4_neighbors
             do_yint = self.do_yintercept
```

Comment:
Can remove these 2 lines, since y_int is being stripped out.

Reply:
removed them.